README.md

```diff
@@ -56,6 +56,46 @@ yarn docker-local
 
 Once you observe 'Listening on :7000' in the log you can see the app by navigating to http://localhost:7000
 
+## Adding Techdocs to your repository
+
+Things you'll need:
+  
+  - [Techdocs-cli](https://github.com/backstage/techdocs-cli)
+  - [S3 credentials](https://docs.aws.amazon.com/AmazonS3/latest/userguide/AuthUsingAcctOrUserCredentials.html)
+  
+Things to do:
+  - In your repo add a mkdocs.yml file similar to the one below.
+
+```yml
+site_name: 'My Repo docs name'
+
+nav:
+  - Home: index.md
+
+plugins:
+  - techdocs-core
+```
+  
+  - Once you have techdocs-cli installed
+  run the techdocs:
+
+```sh
+yarn cli
+```
+  - Then push your credentials to s3
```

Comment:
push the docs to s3, not credentials

Reply:
credentials would be setup with the jenkins pipeline.  I think this is with the role, same as we did for our app, but not sure if that role is available from the jenkins instance.  This might be a road block